metricq_source_bacnet/source.py

```diff
@@ -61,6 +61,7 @@ def __init__(self, *args, disk_cache_filename=None, **kwargs):
         self._worker_tasks: List[Task] = []
         self.disk_cache_filename = disk_cache_filename
         self._old_bacnet_reader_config = {}
+        self._last_value_send_by_metric: Dict[str, Timestamp] = {}
```

Comment:
The name suggests that the dict stores values, but the type annotation says it's `Timestamp`s.